tests/sentry/conf/test_kafka_definitions.py

```diff
@@ -1,10 +1,24 @@
 import pytest
+from django.conf import settings
 
-from sentry.conf.types.consumer_definition import ConsumerDefinition, validate_consumer_definition
+from sentry.conf.types.kafka_definition import (
+    ConsumerDefinition,
+    Topic,
+    validate_consumer_definition,
+)
 from sentry.consumers import KAFKA_CONSUMERS
 from sentry.testutils.cases import TestCase
 
 
+def test_topic_definition() -> None:
```

Comment:
I think we should also have a test to verify that all the topics in `KAFKA_TOPIC_OVERRIDES` are actually in the `Topic` enum.

Reply:
This is always empty/default in dev/ci so it won't do much

Reply:
oh right, I guess this is more for getsentry